tron/bin/recover_batch.py

```diff
@@ -10,6 +10,10 @@
 from twisted.internet import inotify
 from twisted.internet import reactor
 from twisted.python import filepath
+try:
+    from yaml import CSafeLoader as SafeLoader
+except ImportError:
```

Comment:
I don't really want this here.

Can you use `tron.yaml` instead like we do elsewhere in the codebase?

Reply:
Sure.